content/docs/user-guide/analytics.md

```diff
@@ -46,6 +46,7 @@ any execution. They will fail immediately and silently if you have no network
 connection.
 
 DVC's analytics are sent through DVC's proxy to Google Analytics over HTTPS.
+Google Analytics uses IP addresses of these requests for aggregate analytics.
```

Comment:
```suggestion
Google Analytics is aware of user IP addresses, and we are actively looking for [solutions which do not track IPs](https://github.com/iterative/dvc.org/issues/3397).
```

Reply:
The change you made reads like an apology. I'm trying to "mention in a deep footnote." :)

Reply:
I think we do need to somehow mention that soon IPs won't be tracked. Open to any rewording/rephrasing.

Reply:
Make it into a note?

Reply:
This is actually a managerial decision but I'd like to keep it as hidden as possible while writing it. I wouldn't like it to be advertised more than necessary. We don't do anything with those IP addresses at the moment except figuring out the countries where DVC is used. (And those stats show mostly Ireland anyway, and we know that's mostly wrong.) I don't see a need for being apologetic or mentioning that "we'll change this as soon as possible." 

It's about GDPR compliance, and without that, I don't think users would mind or trust what we write here.

Reply:
@shcheklein We need your input here :) Thanks.

Reply:
I think that's enough that we mention that we use GA. That alone gives all the information. 

If we save IP addresses outside of GA let's fix the lambda asap please instead of mentioning it here

Reply:
Checked the source - we don't save IP outside GA.
We only pass IP via `https://www.google-analytics.com/collect?uip=<client_ip>&...`